### PR TITLE
Rdm 2202 Case Mgmt Web to consume new ShowEventNotes

### PR DESCRIPTION
**Pull request by andrewfolga**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-2202

### Change description ###
Case Mgmt Web to consume new ShowEventNotes

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/app/shared/case-editor/case-edit-submit.component.spec.ts [hunk 1]
```diff
@@ -196,6 +196,42 @@ describe('CaseEditSubmitComponent', () => {
...
+  it('should return true when show event notes set to true in event trigger and readOnlySummaryFieldsToDisplayExists is called', () => {
+    comp.eventTrigger.show_event_notes = true;
+    fixture.detectChanges();
+
+    let result = comp.showEventNotes();
+
+    expect(result).toBeTruthy();
+  });
+
+  it('should return false when show event notes not set in event trigger and readOnlySummaryFieldsToDisplayExists is called', () => {
```
Comment (andrewfolga) [inline src/app/shared/case-editor/case-edit-submit.component.spec.ts]: fix test name

File: src/app/shared/case-editor/case-edit-submit.component.spec.ts [hunk 2]
```diff
@@ -196,6 +196,42 @@ describe('CaseEditSubmitComponent', () => {
...
+    comp.eventTrigger.show_event_notes = null;
+    fixture.detectChanges();
+
+    let result = comp.showEventNotes();
+
+    expect(result).toBeTruthy();
+  });
+
+  it('should return false when show event notes not set in event trigger and readOnlySummaryFieldsToDisplayExists is called', () => {
```
Comment (andrewfolga) [inline src/app/shared/case-editor/case-edit-submit.component.spec.ts]: fix test name

File: src/app/shared/case-editor/case-edit-submit.component.ts [hunk 3]
```diff
@@ -157,10 +157,15 @@ export class CaseEditSubmitComponent implements OnInit {
   }
 
   readOnlySummaryFieldsToDisplayExists(): boolean {
-    // found no fields to show in read only summary page
     return this.eventTrigger.case_fields.some(field => field.show_summary_content_option >= 0 );
   }
 
+  showEventNotes(): boolean {
+    return this.eventTrigger.show_event_notes
```
Comment (mario-paniccia) [inline src/app/shared/case-editor/case-edit-submit.component.ts]: could this be simplified with:

.show_event_notes !== false

File: src/app/shared/case-editor/case-edit-submit.html [hunk 4]
```diff
@@ -65,20 +65,22 @@ <h2 class="plain-message">Check the information below carefully. This will form
...
-      </label>
-      <input type="text" id="field-trigger-summary" class="form-control" formControlName="summary" maxlength="1024">
-    </div>
-    <div class="form-group">
-      <label for="field-trigger-description" class="form-label">Event description (optional)</label>
-      <textarea id="field-trigger-description" class="form-control" formControlName="description"
-                maxlength="65536"></textarea>
-    </div>
-  </fieldset>
+  <ng-container *ngIf="showEventNotes()">
```
Comment (mario-paniccia) [inline src/app/shared/case-editor/case-edit-submit.html]: I think we should to add a unit test for this template to verify the this whole block is not displayed if showEventNotes() if false

Comment (andrewfolga) [reply src/app/shared/case-editor/case-edit-submit.html]: Done.

**Pull request merged**